test/Interpreter/SDK/Foundation_NSString.swift

```diff
@@ -136,21 +136,27 @@ testComparisons()
 // CHECK-LABEL: Variadic methods:
 print("Variadic methods:")
 // CHECK-NEXT: x y
-print(NSString(format: "%@ %@", "x" as NSString, "y" as NSString))
+print(NSString(format: "%@ %@", "x", "y"))
+// CHECK-NEXT: (
+// CHECK-NEXT:   x
+// CHECK-NEXT: ) {
+// CHECK-NEXT:   y = z;
+// CHECK-NEXT: }
+print(NSString(format: "%@ %@", ["x"], ["y": "z"]))
```

Comment:
Please add a better, more descriptive test or at least a comment about what is important about this line.  It is not clear what this line is testing.